.travis.yml

```diff
@@ -2,8 +2,14 @@ language: python
 python:
   - "2.7"
   - "3.4"
+env:
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+  - DJANGO_VERSION=1.9b1
 install:
   - pip install -r dev-requirements.txt
+  - pip uninstall django --yes
+  - pip install -q django==$DJANGO_VERSION --pre
```

Comment:
that's may be right. i added when i put `1.9` for version. but now as I've specified the version as `1.9b1`, this may be optional.